primitives/npos-elections/compact/tests/compact.rs

```diff
@@ -0,0 +1,168 @@
+use std::{
```

Comment:
needs header

Reply:
Currently refactoring to add a benchmark; this file is being entirely removed and split into other files.